### ISSUE TITLE
Seems impossible to turn on profiling with the Chez compiler

### ISSUE THREAD

**Issue by redfish64** [idris-lang/Idris2#1572]
The Chez compiler will silently ignore it if you turn the profile flag on and run a command using ":exec" within the repl or ide.
Also, if you specify "--profile" as a command line argument, along with "--build", it responds with:
```
Not all command line options can be used to override package options.
```
and there doesn't seem to be a option for setting "profile" within the ipkg

**Comment by redfish64** [comment]
Added a pull request: https://github.com/idris-lang/Idris2/pull/1573

**Comment by CodingCellist** [comment]
Closed by #1576?  : )
